lib/suspenders/generators/app_generator.rb

```diff
@@ -226,6 +227,11 @@ def copy_miscellaneous_files
       build :copy_miscellaneous_files
     end
 
+    def download_style_guides
+      say 'Downloading style guides from thoughtbot/guides'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.